app/models/qernel/reconciliation/transformation_adapter.rb

```diff
@@ -21,12 +21,16 @@ def setup(phase:)
         end
       end
 
+      def carrier_demand
+        carrier_demand_input - carrier_demand_output
+      end
+
```

Comment:
@mabijkerk can you verify this? Or should this value always be the net difference (so with an `abs` around it)?